webroot/js/components/player.js

```diff
@@ -79,6 +82,9 @@ class OwncastPlayer {
   startPlayer() {
     this.log('Start playing');
     const source = { ...VIDEO_SRC };
+
+    if (getLocalStorage(PLAYER_VOLUME) !== null)
```

Comment:
Hey @SoldierCorp , Thanks for doing this! Looks great. 
Just a small nit - can we put `{}` around this condition, just to keep the code style consistent across the app? :)

thanks!

Reply:
I added a cleaner version